tests/dummy_modules/llama-seq-classification-tiny/tokenizer_config.json

```diff
@@ -138,5 +138,6 @@
   "sep_token": "<sep>",
   "sp_model_kwargs": {},
   "tokenizer_class": "T5Tokenizer",
-  "unk_token": "<unk>"
+  "unk_token": "<unk>",
+  "chat_template": "{%- for message in messages %}\n    {%- if message['role'] == 'user' %}\n        {{- '<|user|>' + message['content'] + eos_token -}}\n    {%- elif message['role'] == 'system' %}\n        {{- '<|system|>' + message['content'] + eos_token -}}\n    {%- elif message['role'] == 'assistant' %}\n        {{- '<|assistant|>' + message['content'] + eos_token }}\n    {%- endif %}\n{%- if loop.last and add_generation_prompt -%}\n  {{- '<|assistant|>' -}}\n{%- endif -%}\n{%- endfor %}"
```

Comment:
With latest `transformers`, we need to set chat_template to use it.
(Older versions automatically set the default template, but not now)

Reply:
[memo] chate_template of llama-seq-classification model is used in here
https://github.com/sbintuitions/flexeval/blob/aa1c87d6a94bcafc32e16842954cd60f3b5b93d8/flexeval/core/reward_model/sequence_classification.py#L39
while executing this test
https://github.com/sbintuitions/flexeval/blob/aa1c87d6a94bcafc32e16842954cd60f3b5b93d8/tests/core/reward_model/test_sequence_classification.py#L12